packages/workflow/src/constants.ts

```diff
@@ -30,6 +30,7 @@ export const CERT_PUBLIC_KEY_PATH =
   '../../.secrets/public-key.pem'
 export const DOCUMENTS_URL =
   process.env.DOCUMENTS_URL || 'http://localhost:9050'
+export const MINIO_URL = process.env.MINIO_URL ?? 'localhost:3535'
```

Comment:
docker-compose file needs the env variable

Reply:
I think it won't be needed if we change the check for `presignedUrl` like Riku suggested